.travis.yml

```diff
@@ -16,7 +16,7 @@ matrix:
   #  - we insist on rust 1.14 only for Debian, and
```

Comment:
Is that comment up to date? doesn't seem like we're using rand 0.4, nor insisting on rust 1.14